build.gradle

```diff
@@ -254,7 +257,7 @@ def versions = [
         pitest              : '1.6.3',
         sonarPitest         : '0.5',
         serviceAuthVersion  : '4.0.0',
-        springFramework     : '5.3.9'
+        springFramework     : '5.3.7'
```

Comment:
Does this need to be downgraded?

Reply:
Hi Dave , before any changes it was 5.3.7. So last time when we did the change we upgraded it . Now I put it back same as pcq-backend 5.3.7.

Reply:
Hi Dave, springFrameowrk has been moved back to 5.3.9. and nightly-dev build is geeen . Please see attached screen shot.
<img width="1670" alt="Screenshot 2021-08-19 at 14 35 24" src="https://user-images.githubusercontent.com/88436206/130078125-bbca7424-6155-4343-b399-3358ad030e42.png">